snippets/country-localization.liquid

```diff
@@ -31,11 +33,11 @@
             {% endif %}
             data-value="{{ country.iso_code }}"
           >
-            <span class="localization-form__currency">
-              {{- country.currency.iso_code }}
-              {{ country.currency.symbol }} |</span
-            >
-            {{ country.name }}
+            <span class="localization-form__disclosure__item-wrapper">
```

Comment:
```suggestion
            <span class="localization-form__disclosure-item-wrapper">
```

Reply:
changed it to follow the BEM naming convention! WIll be seen in the next commit!